src/openzaak/import_data/tasks.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2024 Dimpact
+import logging
+from datetime import timedelta
+
+from django.utils import timezone
+
+from openzaak import celery_app
+from openzaak.import_data.models import Import, ImportStatusChoices
+
+logger = logging.getLogger(__name__)
+
+
+@celery_app.task()
+def remove_imports(days_back=7):
```

Comment:
let's make `days_back` a django setting which can be configured with an env var. Since it's a celery beat task I don't think we will call it with parameters